integrations/repo_fork_test.go

```diff
@@ -5,19 +5,20 @@
 package integrations
 
 import (
+	"fmt"
 	"net/http"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
-func testRepoFork(t *testing.T, session *TestSession) *TestResponse {
+func testRepoFork(t *testing.T, session *TestSession, ownerName, repoName, forkOwnerName, forkOwnerID, forkRepoName string) *TestResponse {
```

Comment:
nit: instead of requiring caller to specify `forkOwnerID`, we can infer it from `forkOwnerName`:
```
forkOwner := AssertExistsAndLoadBean(t, &models.User{Name: forkOwnerName}).(*models.User)
...
forkOwner.ID
```

Reply:
@ethantkoenig thanks, done